lib/Notifications.dart

```diff
@@ -0,0 +1,208 @@
+                        style: TextStyle(fontSize: 15, color: Colors.black),
+                      ),
+                    ),
+                  ],
+                ),
+                SizedBox(height: 15),
+                Row(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    CircleAvatar(
+                      radius: 22,
+                      child: CircleAvatar(
+                        radius: 22,
+                        backgroundColor: Colors.black,
+                        foregroundColor: Colors.white,
+                        child: Icon(Icons.check),
+                      ),
+                    ),
+                    SizedBox(width: 19),
+                    Expanded(
+                      child: Text(
+                        "Your ride on MM/DD/YYYY has been confirmed.",
+                        style: TextStyle(fontSize: 15, color: Colors.black),
+                      ),
+                    ),
+                  ],
+                ),
+                SizedBox(height: 15),
+                Row(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    CircleAvatar(
+                      radius: 22,
+                      child: CircleAvatar(
+                        radius: 22,
+                        backgroundColor: Colors.black,
+                        foregroundColor: Colors.white,
+                        child: Icon(Icons.directions_car),
+                      ),
+                    ),
+                    SizedBox(width: 19),
+                    Expanded(
+                      child: Text(
+                        "How was your ride with Jennifer? Leave a review so we can improve our service.",
+                        style: TextStyle(fontSize: 15, color: Colors.black),
+                      ),
+                    ),
+                  ],
+                ),
+                SizedBox(height: 15),
+                Row(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    CircleAvatar(
+                      radius: 22,
+                      child: CircleAvatar(
+                        radius: 22,
+                        backgroundColor: Colors.red,
+                        foregroundColor: Colors.white,
+                        child: Icon(Icons.edit),
+                      ),
+                    ),
+                    SizedBox(width: 19),
+                    Expanded(
+                      child: Text(
+                        "Your ride information has been edited by the admin. Please review your ride info.",
+                        style: TextStyle(fontSize: 15, color: Colors.black),
+                      ),
+                    ),
+                  ],
+                ),
+                SizedBox(height: 15),
```

Comment:
Maybe refactor this whole column into a list view of some sort? Then you can set a divider between list elements instead of copying and pasting SizedBoxes. This will also make it easier to integrate everything with the backend later on.